### PR TITLE
Add climate support

### PR DESCRIPTION
**Pull request by Necroneco**
Try adding `climate` support, but not tested.

### CODE REVIEW

File: custom_components/terncy/const.py [hunk 1]
```diff
@@ -27,7 +27,8 @@
 PROFILE_YAN_BUTTON = 6
 PROFILE_SMART_DIAL = 7
 PROFILE_COLOR_LIGHT = 8
-
+# PROFILE_9 = 9  # 中央空调控制器
+PROFILE_10 = 10  # 中央空调分机
```
Comment (rxwen) [inline custom_components/terncy/const.py:31]: Suggest rename to PROFILE_AC_UNIT_MACHINE

File: custom_components/terncy/profiles/before_2023_7.py [hunk 2]
```diff
@@ -6,7 +6,9 @@
 from homeassistant.helpers.entity import EntityCategory  # <2023.3
 
 from ..binary_sensor import TerncyBinarySensorDescription
+from ..climate import TerncyClimateDescription
 from ..const import (
+    PROFILE_10,
     PROFILE_11_LOCK,
     PROFILE_18,
```
Comment (rxwen) [inline custom_components/terncy/profiles/before_2023_7.py:13]: Suggest rename to PROFILE_OCCUPANCY_SENSOR

**Review by rxwen — CHANGES_REQUESTED** [review]
suggest update some naming of const

File: custom_components/terncy/const.py (hunk 1 above)
Comment (Necroneco) [reply custom_components/terncy/const.py]: ok, renamed

File: custom_components/terncy/profiles/before_2023_7.py (hunk 2 above)
Comment (Necroneco) [reply custom_components/terncy/profiles/before_2023_7.py]: Renamed PROFILE_18 to PROFILE_OCCUPANCY_SENSOR

**Pull request merged**